src/pages/Sketch/components/Canvas.js

```diff
@@ -109,11 +110,12 @@ function Canvas() {
         context.lineCap = 'round';
         context.lineWidth = width;
 
-        if(type === 'pen') {
+        if(type === 'pen'|| type==='eraser') { //if type is pen then logicDown function is called
```

Comment:
```suggestion
        //if type is pen then logicDown function is called
        if(['pen', 'eraser'].includes(type)) { 
```
DO this instead. Please note: it's better to add comments prior to the actual logic. (update it where ever required.)